pkg/operator/osd/osd.go

```diff
@@ -107,8 +107,15 @@ func (c *Cluster) Start() error {
 
 			resources := k8sutil.MergeResourceRequirements(c.Storage.Nodes[i].Resources, c.resources)
 
+			var directories []Directory
```

Comment:
This is more like a noob go question. Wouldn't it be possible to delete the initialization of the directories variable above, and just use the `:=` operator ? (which already does the initialization).

Reply:
In the `if` condition it is a new scope, so if I define/initialize the variable in the `if` the variable isn't available outside the `if` condition, that is why I initialize the variable on line 110.